index.html

```diff
@@ -5,310 +5,224 @@
-                    <p class="contact-description-1">
-                        If you have an application you are interested in developing, a
-                        feature that you need built or a project that needs coding. I’d
-                        love to help with it.
-                    </p>
+        </div>
+    </section>
+    <section class="about-container">
+        <div class="top">
+            <div class="about-title">About <br>
+                Myself</div>
+            <div class="about-content">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                contact me.
+            </div>
+            <div class="about-sm">
+                <h4 class="supporting-text">LET’S CONNECT</h4>
+                <ul class="about-smedia-icons">
+                    <li><img src="pics/github icon.svg" alt="Github"></li>
+                    <li><img src="pics/Linkedin icon.svg" alt="Linkedin"></li>
+                    <li><img src="pics/angellist icon.svg" alt="Angellist"></li>
+                    <li><img src="pics/twitter icon.svg" alt="Linkedin"></li>
+                    <li><img src="pics/m icon.svg" alt="Med"></li>
+                </ul>
+            </div>
+            <button class="btn2">Get my resume</button>
+        </div>
+        <div class="experties">
+            <ul class="skill-list">
+                <li class="languages">
+                    <span class="Lang-title">Languages</span>
+                    <i class="fa fa-angle-down" style="font-size: 24px"></i>
+                </li>
+                <ul class="lang-list">
+                    <li class="lang">
+                        <img src="pics/works img/Icons/js icon.svg" alt="">
+                        <span>JavaScript</span>
+                    </li>
+                    <li class="lang">
+                        <img src="pics/works img/Icons/html icon.svg" alt="">
+                        <span>HTML</span>
+                    </li>
+                    <li class="lang">
+                        <img src="pics/works img/Icons/css icon.svg" alt="">
+                        <span>CSS</span>
+                    </li>
+                </ul>
+                <li class="languages border">
+                    <span class="Lang-title">Framework</span>
+                    <i class="fa fa-angle-right" style="font-size: 24px"></i>
+                </li>
+                <li class="languages border">
+                    <span class="Lang-title">Skills</span>
+                    <i class="fa fa-angle-right" style="font-size: 24px"></i>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="form-section">
+        <div class="background">
+            <div class="form">
```

Comment:
- [OPTIONAL]  I would suggest not giving tag names like `form` as class names. This can be confusing at some point. This is because you can as well use tag names for styling. Whats your take 🤔 ?